index.bs

```diff
@@ -1693,6 +1768,122 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The browsingContext.navigate Command ####  {#command-browsingContext-navigate}
+
+The <dfn export for=commands>browsingContext.navigate</dfn> command navigates a
+browsing context to the given URL.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextNavigateCommand = {
+        method: "browsingContext.navigate",
+        params: BrowsingContextNavigateParameters
+      }
+
+      BrowsingContextNavigateParameters = {
+        context: BrowsingContext,
+        url: text,
+        ?wait: ReadinessState,
```

Comment:
To be honest I don't know. @mathiasbynens can you help us figure out why Puppeteer has this API with an array of things to wait for? Is it because the events really don't have an inherent order, or is it some kind of convenience for call sites that maybe are waiting for "networkidle0" plus an event given as an argument, so that they don't have to figure out how to collapse two things into one?